android/modules/ui/src/js/listview.js

```diff
@@ -115,7 +115,14 @@ exports.bootstrap = function (Titanium) {
 					// widget does not exist, attempt to load namespace
 					widget = Module.main.require(namespace);
 				} catch (err) {
-					// do nothing...
+					// namespace does not exist, fall back to legacy behaviour
+					let parentProxy = global;
+					for (let i = 0; i < segments.length; i++) {
+						parentProxy = parentProxy[segments[i]];
+					}
+					if (parentProxy) {
+						return parentProxy['create' + proxyName];
```

Comment:
<!--
  1 failure:  android/modules/u...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: android/modules/ui/src/js/listview.js;
  Line: 124;
-->
- :no_entry_sign: android/modules/ui/src/js/listview.js line 124 – 'proxyName' is not defined. (no-undef)